queries.sql

```diff
@@ -7,4 +7,133 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' or name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
+
+-- update the animals table by setting the species column to unspecified
+UPDATE animals
+SET species = 'unspecified';
+
+-- Rollback database
+Rollback;
+
+-- inside a transaction
+BEGIN;
+
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species != 'digimon';
+
+-- Commit the transaction.
+
```

Comment:
Please add query before commit as well to verify explicitly that change was made.

Reply:
resolved